Godeps/Godeps.json

```diff
@@ -89,6 +89,10 @@
 			"ImportPath": "github.com/armPelionEdge/zeroconf",
 			"Rev": "78d055b6305c685b497944a859f0a7d14483269f"
 		},
+		{
```

Comment:
How is this file used in conjunction with the vendor tree?

Last I checked (a while back, admittedly), this file was not in sync with the rest of the vendor contents.

Reply:
Hmm, The current change is to sync with latest maestroSpecs. May be we can separately review the integrity of this file.

Reply:
Fair. Would you mind creating an issue for it?

Reply:
Note for future Cruz: Internal Jira ticket is still open, since this isn't the only GoLang project affected by the Godeps.json/vendor issue.